server/repository/src/db_diesel/stock_line.rs

```diff
@@ -121,11 +146,15 @@ impl<'a> StockLineRepository<'a> {
     }
 }
 
-type BoxedStockLineQuery =
-    IntoBoxed<'static, LeftJoin<LeftJoin<stock_line::table, location::table>, name::table>, DBType>;
+type BoxedStockLineQuery = IntoBoxed<
+    'static,
+    LeftJoin<LeftJoin<InnerJoin<stock_line::table, item::table>, location::table>, name::table>,
```

Comment:
I couldn't see a way to order the results by the result of the item loader - so I've just joined the item table to the stock line 🤷